ReactiveCocoa/NSObject+RACDescription.m

```diff
@@ -48,3 +48,15 @@ - (NSString *)rac_description {
 }
 
 @end
+
+NSString *rac_description(id object) {
+	if ([object respondsToSelector:@selector(rac_description)]) {
```

Comment:
Everything `id` has to obey the `<NSObject>` protocol—which includes a `-class` method—as a matter of practicality. AFAIK Swift objects are included in that regard.